### DISCUSSION TITLE
When I pull or fetch from the remote origin to my local pc I get a fatal error

### DISCUSSION THREAD

**Discussion by youssefantoury** [community/community#109739] [Pull Requests]
### Select Topic Area

Bug

### Body

Hello Everyone, 

I made a lot of changes on my 1st device and pushed everything, when I try to pull them from my 2nd device on Github Desktop I get the following error message:

remote: Enumerating objects: 92, done.        
remote: Counting objects: 100% (92/92), done.        
remote: Compressing objects: 100% (76/76), done.        
error: RPC failed; curl 18 transfer closed with outstanding read data remaining
error: 8079 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: unpack-objects failed

I tried from the command line and it's still the same. The Internet connection is also stable. And I'm running the last version of the app.
This hasn't happened before, maybe pullling in another folder might work but there is a lot of data in the 1st one so I prefer to not have redundant data due to some error that I don't know the cause of yet.

Thank you in advance, 
Youssef

**Comment by muspriandi** [comment]
Hi Youssef...
This problem may happen when the repository is big. Your computer might have trouble downloading all the data from the server, even if your internet connection is good.

Maybe you can try increasing Git’s buffer size with this command:
```
git config --global http.postBuffer 524288000
``` 
Then try pull or fetch again.

Note that 524288000 is configurable → means 500 MB (that’s the new limit)
